templates/hotgridPopup.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useEffect } from 'react';
+import a11y from 'core/js/a11y';
+import { classes, templates } from 'core/js/reactHelpers';
+
+export default function HotgridPopup(props) {
+
+  const {
+    _items,
+    _isRound
+  } = props;
+
+  useEffect(() => {
+    const activeItem = _items.filter(item => item._isActive);
+
+    if (!activeItem.length) return;
+
+    const activeItemIndex = activeItem[0]._index;
+    a11y.focusFirst($(`.hotgrid-popup__item[data-index=${activeItemIndex}]`));
+  });
+
+  return (
+    <div className='hotgrid-popup__inner'>
+
+      {_items.map(({ title, body, _itemGraphic, _graphic, _classes, _isVisited, _isActive, _imageAlignment }, index) =>
+        <div className={classes([
+          'hotgrid-popup__item',
+          _classes,
+          _isRound && 'is-round',
+          _isVisited && 'is-visited',
+          _isActive && 'is-active',
+          _imageAlignment && `align-image-${_imageAlignment}`
+        ])}
+        key={index}
+        data-index={index}
+        aria-hidden={!_isActive && null}
```

Comment:
Rather than render out aria-hidden='false' when its not hidden can we just not have that in that case pls? So only renders out aria-hidden=true when true pls

Reply:
Fixed. I had this logic backwards.